resource.language.ro_ro/addon.xml

```diff
@@ -20,7 +20,9 @@
   </extension>
   <extension point="kodi.addon.metadata">
     <summary lang="en">Romanian language pack</summary>
+    <summary lang="ro">Pachet lingvistic pentru limba română</summary>
```

Comment:
I believe there is no sense translating these texts, as they will only be visible when the addon is already loaded so only the English text will be used 99.999999% of the times.
@MartijnKaijser What is your opinion ?